src/main/java/gov/usgs/wma/waterdata/data/DataController.java

```diff
@@ -0,0 +1,79 @@
+package gov.usgs.wma.waterdata.data;
+
+import gov.usgs.wma.waterdata.OgcException;
+import gov.usgs.wma.waterdata.collections.BaseController;
+import gov.usgs.wma.waterdata.collections.CollectionParams;
+import gov.usgs.wma.waterdata.openapi.schema.timeseries.TimeSeriesGeoJSON;
+import gov.usgs.wma.waterdata.parameter.ContentType;
+import gov.usgs.wma.waterdata.parameter.DataType;
+import gov.usgs.wma.waterdata.parameter.Domain;
+import gov.usgs.wma.waterdata.timeseries.TimeSeriesDao;
+import io.swagger.v3.oas.annotations.ExternalDocumentation;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.enums.ParameterIn;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.HttpMediaTypeNotAcceptableException;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Observations Data Sets", description = "Data sets such as Statistical Time Series")
```

Comment:
Might ask Jim what his concept of 'data' vs 'collection' is.  I think the 'data' stuff doesn't fit neatly into collections, but I'm not sure how.